lib/reservation_tracker.rb

```diff
@@ -0,0 +1,171 @@
+require_relative 'lib_file_helper'
+
+class ReservationTracker
+  attr_accessor :all_rooms, :all_reservations
+
+  def initialize()
+    # @all_rooms is a collection of all room instances
+    @all_rooms = room_maker
+    # @all_reservations is a collection of all reservation instances
+    @all_reservations = []
+  end
+
+  def room_maker
```

Comment:
I like naming objects, classes, etc as _things_, and methods as _actions_ ... instead of calling this `room_maker`, I might suggest calling it `make_rooms` instead?